templates/kube-controller-manager.yaml

```diff
@@ -20,6 +20,9 @@ spec:
     - --service-account-private-key-file=/etc/kubernetes/ssl/apiserver-key.pem
     - --root-ca-file=/etc/kubernetes/ssl/ca.pem
     - --leader-elect=true
+{% if controllers is defined %}
+    - --controllers={{controllers}}
```

Comment:
I hit Approve too soon. Can we please change the var to `kube_controllers` and add a default to `defaults/main.yml`?

Reply:
I changed the var to `kube_controllers`.
We don't want to set a default value for `kube_controllers` because, by default, we want kube-controller-manager to run with the default set of controllers listed here https://kubernetes.io/docs/reference/command-line-tools-reference/kube-controller-manager/

Reply:
I was just thinking from the Ansible Role perspective. It's better to list all possible variables in defaults/main.yml.

Reply:
Fixed now.